Makefile

```diff
@@ -87,3 +87,13 @@ dashboards-lint: $(GRAFANA_DASHBOARD_LINTER_BIN) $(OUT_DIR)/.lint
 	# Replace $$interval:$$resolution var with $$__rate_interval to make dashboard-linter happy.
 	@sed -i -e 's/$$interval:$$resolution/$$__rate_interval/g' $(OUT_DIR)/*.json
 	@find $(OUT_DIR) -name '*.json' -print0 | xargs -n 1 -0 $(GRAFANA_DASHBOARD_LINTER_BIN) lint --strict
+
+.PHONY: test
+test: test-jsonnet
+
+.PHONY: test-jsonnet
+test-jsonnet: $(JSONNET_BIN) $(JSONNET_VENDOR)
+	@echo "Running jsonnet query tests..."
+	@$(JSONNET_BIN) -J vendor tests/pod_queries_test.libsonnet > /dev/null
+	@$(JSONNET_BIN) -J vendor tests/namespace_queries_test.libsonnet > /dev/null
+	@echo "All tests passed!"
```

Comment:
Sanity check - does this always output `All tests passed!` or only if it gets this far without failure?